CMakeLists.txt

```diff
@@ -149,9 +146,13 @@ endif()
 target_compile_definitions(fms PUBLIC "${_fms_defs_public}")
 
 if(32BIT)
+  set(DYN32 ON CACHE BOOL "Enable support for 32bit fast physics in CCPP")
```

Comment:
Does this need to be in the top-level `CMakeLists.txt`?
CCPP is not used with `DATM=ON`.  We should move this to `FV3/CMakeLists.txt`, unless `DYN32` is used elsewhere.